### PR TITLE
NativeFunctionCasingFixer - call to constructor with default NS of class with name matching native function name fix

### PR DESCRIPTION
**Pull request by SpacePossum**
…prefix by default namespace.

replaces https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2048 on 1.12 this time

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:+1:

File: Symfony/CS/Fixer/Symfony/NativeFunctionCasingFixer.php [hunk 1]
```diff
@@ -50,12 +50,12 @@ public function fix(\SplFileInfo $file, $content)
...
             }
 
-            // do not though the function call if it is to a function in a namespace other than the default
-            if (
-                $tokens[$functionNamePrefix]->isGivenKind(T_NS_SEPARATOR)
-                && $tokens[$tokens->getPrevMeaningfulToken($functionNamePrefix)]->isGivenKind(T_STRING)
-            ) {
-                continue;
+            if ($tokens[$functionNamePrefix]->isGivenKind(T_NS_SEPARATOR)) {
+                // do not touch the function call if it is to a function in a namespace other than the default
```
Comment (keradus) [inline Symfony/CS/Fixer/Symfony/NativeFunctionCasingFixer.php]: the comment is outdated now, as we also check class, not only function now

**Comment by keradus** [comment]
:+1: but comment

**Comment by SpacePossum** [comment]
updated, how to squash tomorrow, thanks for the review :)

**Comment by GrahamCampbell** [comment]
Good for merging now, right @keradus?

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**